app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -732,44 +730,50 @@ protected void setupGUI() {
         // BUTTON SETUP
 
         if(butPlaybackSpeed != null) {
-            butPlaybackSpeed.setOnClickListener(v -> {
-                if (controller == null) {
-                    return;
-                }
-                if (controller.canSetPlaybackSpeed()) {
-                    String[] availableSpeeds = UserPreferences.getPlaybackSpeedArray();
-                    String currentSpeed = UserPreferences.getPlaybackSpeed();
-
-                    // Provide initial value in case the speed list has changed
-                    // out from under us
-                    // and our current speed isn't in the new list
-                    String newSpeed;
-                    if (availableSpeeds.length > 0) {
-                        newSpeed = availableSpeeds[0];
-                    } else {
-                        newSpeed = "1.00";
```

Comment:
This allows, for now, the same layout to be used to cast or local audio player activities